Packs/Okta_v2/TestPlaybooks/playbook-OktaV2-Test.yml

```
@@ -1329,37 +1100,114 @@ tasks:
       brand: Okta v2
     nexttasks:
       '#none#':
-      - "50"
+      - '50'
     scriptarguments:
       limit:
-        simple: "1"
+        simple: '1'
       since: {}
       sortOrder: {}
       until: {}
     separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 695,
-          "y": 2470
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 1452.5,\n    \"y\": 2995\n  }\n}"
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
-view: |-
-  {
-    "linkLabelsPosition": {},
-    "paper": {
-      "dimensions": {
-        "height": 3010,
-        "width": 1885,
-        "x": 50,
-        "y": 50
-      }
-    }
-  }
+  '54':
+    id: '54'
+    taskid: bbdc4cf5-914f-4169-8777-3e0a04cb566c
+    type: regular
+    task:
+      id: bbdc4cf5-914f-4169-8777-3e0a04cb566c
+      version: -1
+      name: Check if Test1@test.com exists
+      description: Searches for Okta users.
+      script: Okta v2|||okta-search
```

better run the command without a brand
```suggestion
      script: '|||okta-search'
```